docs/UserGuide.md

```diff
@@ -219,7 +241,16 @@ _Details coming soon ..._
 ## Known issues
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
+2. **View -Stats Command**, Currently the total fee owing by students can precision more than 2 decimal points
```

Comment:
Not sure if `View -Stats` should be standardized to `view -stats` for consistency in case!